.coderabbit.yaml

```
@@ -0,0 +1,133 @@
+language: en-US
+tone_instructions: ''
+early_access: false
+enable_free_tier: true
+reviews:
+  profile: assertive
+  request_changes_workflow: false  # Approve the review once CodeRabbit's comments are resolved. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)